radis/lbl/factory.py

```diff
@@ -476,7 +476,12 @@ def __init__(
         # Initialize computation variables
         self.params.wstep = wstep
         self.params.pseudo_continuum_threshold = pseudo_continuum_threshold
-        self.params.cutoff = cutoff
+
+        if cutoff is None:
+            self.params.cutoff = 0
+        else:
+            self.params.cutoff = cutoff
+
```

Comment:
```suggestion
        if cutoff is None:
            # If None, use no cutoff : https://github.com/radis/radis/pull/259
            cutoff = 0
        self.params.cutoff = cutoff
```

Just a suggestion that shows better we're re-writing the default option (also good practice in case "cutoff" is ever used in the same function, not to have different values for "cutoff" and "self.params.cutoff")